backend_jobs/recognition_pipeline/filters/filter_by_resolution.py

```diff
@@ -26,6 +26,8 @@ class FilterByResolution(FilterBy):
     def is_supported(self, image):
```

Comment:
The filerbyresolution class does more than filtering now - it also enriches the dictionary. Consider adding another phase to avoid this semantic mix. One phase should add the URL, and the other phase should filter out (if URL is not provided)

Reply:
devided into 2 different methods - 1. add field 2. is supported